game/templates/game/landing.html

```
@@ -112,6 +112,9 @@
         </div>
       </div>
       <div class="nav-right">
+        <button id="themeToggle" class="theme-toggle">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)